backend/config/routes.rb

```diff
@@ -168,9 +168,12 @@
       member do
         get :orders
         get :items
-        get :addresses
```

Comment:
I still have a concern related to backward compatibility related to these routes changes: am I right that the route helper method `addresses_admin_user_path` is no more available (it's called `admin_user_addresses_path` now)? If yes, I think we should keep it somehow.